Default.sublime-commands

```diff
@@ -67,6 +67,8 @@
     { "caption": "Toggle Comment", "command": "toggle_comment", "args": {"block": false} },
     { "caption": "Toggle Block Comment", "command": "toggle_comment", "args": {"block": true} },
 
+    { "caption": "Wrap at Ruler", "command": "wrap_lines" },
+
```

Comment:
Already added it earlier, but forgot to push. It's now in master.

Reply:
Do you want me to remove this line from the pull request?

Reply:
It would add a duplicated item when merged, so yes.

Reply:
done!